app/views/ConfirmationView.scala.html

```diff
@@ -88,36 +88,35 @@ <h2 class="heading-small">@messages("confirmation.scotlandHeading")</h2>
         <h2>@messages("confirmation.whatHappensNext")</h2>
 
         @if(!taxYearSelections.contains(TaxYearSelection.CurrentYear)) {
-            <p>@messages("confirmation.letterConfirmation")</p>
+            <p>@messages("confirmation.firstConfirmationLetter")</p>
```

Comment:
pedantic but can we give context to the confirmation letter name, so instead of first and second use current year and previous year?